proknow-sdk/Requestor.cs

```diff
@@ -300,7 +301,8 @@ internal async Task<string> GetDomainStatusAsync()
             string hasMore = "true";
             string next = null;
             var patients = new List<string>();
-            var isPatientsRoute = route.StartsWith("/collections") ? false : true;
+            var match = Regex.Match(route, @"/workspaces/.+/patients");
```

Comment:
You can make this tighter by using the `^` and `$` operators to ensure this is the full string:
```suggestion
var match = Regex.Match(route, @"^/workspaces/.+/patients$");
```
Might need to update the regex expression if the route actually starts with `/api`.